docs/modules/ROOT/pages/ai-services.adoc

```diff
@@ -113,6 +113,14 @@ TriagedReview triage(String review);
 
 In this instance, Quarkus automatically creates an instance of `TriagedReview` from the LLM's JSON response.
 
+To enhance the flexibility of prompt creation, the `{response_schema}` placeholder can be used within the prompt. This placeholder is dynamically replaced with the defined schema of the method's return object.
+
+[IMPORTANT]
+====
+By default, if the placeholder `{response_schema}` is not present in `@SystemMessage` or `@UserMessage`, it will be added to the end of the prompt.
```

Comment:
```suggestion
By default, if the placeholder `+{response_schema}+` is not present in `@SystemMessage` or `@UserMessage`, it will be added to the end of the prompt.
```

dtto as in https://github.com/quarkiverse/quarkus-langchain4j/pull/699/files#r1650503191

Reply:
It seems that one solution to avoid the warning is to use `\{response_schema\}`.